docs/commands/unbind.md

```diff
@@ -16,6 +16,7 @@ Deletes a service binding.
 |--------|-----------|
 | -h, --help  Help for unbind command.| No |
 | -f, --force Force delete without confirmation | No |
+| --purge Delete this resource and all its related data | No |
```

Comment:
same here..

Reply:
Changed to --force-delete with description by Daniel Korac